tests/foreman/cli/test_subscription.py

```diff
@@ -228,3 +228,24 @@ def test_negative_manifest_refresh(self):
             Subscription.refresh_manifest({
                 'organization-id': self.org['id'],
             })
+
+    @skip_if_bug_open('bugzilla', 1686916)
+    @tier2
+    def test_positive_subscription_list(self):
+        """Verify that subscription list contains start and end date
+
+        :id: 4861bcbc-785a-436d-98ce-14cfef7d6907
+
+        :expectedresults: subscription list contains the start and end date
+
+        :BZ: 1686916
+
+        :CaseImportance: Medium
+        """
+        self._upload_manifest(self.org['id'])
+        subscription_list = Subscription.list(
```

Comment:
```suggestion
        subscriptions_list = Subscription.list(
```

Reply:
I think subscription_list is more valid here as the list is not single entity. It bound to have multiple objects  e.g. I can not write like repositories_list or products_list same manner

Reply:
hummm ...it's fine then :)